pillar/fluentd/server_operations_qa.sls

```diff
@@ -95,10 +89,11 @@ fluentd:
                 directive_arg: '**'
                 attrs:
                   - '@id': es_outbound
-                  - '@type': elasticsearch_dynamic
+                  - '@type': elasticsearch
+                  - scheme: https
```

Comment:
Will this work? I don't believe the logging ES hosts are running as HTTPS.

Reply:
I was hoping that we'd leverage the pki infra for use on those ES hosts and thus be able to use `https`.